channels.yaml

```diff
@@ -301,5 +307,11 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
+  - version: v1.24.11-rc1+k3s1
+    minChannelServerVersion: v2.6.7-alpha1
+    maxChannelServerVersion: v2.7.99
+    serverArgs: *serverArgs-v3
+    agentArgs: *agentArgs-v2
+    featureVersions: *featureVersions-v1
   #  v1.25.1+k3s1 was never released
   #  v1.25.2+k3s1 is not released through KDM yet
```

Comment:
I'm surprised there are no 1.25 versions in `dev-v2.7` for k3s (but there are for RKE2 - so this doesn't make sense).

Reply:
Update: Looks like rebase is needed and we do want to add 1.25.7 here (actually replacing any not-yet-released 1.25.x versions).

Reply:
Thats a good catch - @matttrach since 2.7.2 will support Kubernetes 1.25 we should start adding the 1.25 releases to KDM. We're doing that for RKE2 starting with https://github.com/rancher/kontainer-driver-metadata/pull/1099/files#diff-a0456795a97052d90c30065cde4425e798e52b4a119590fc8a7e66416ec5ce61R1092

Reply:
done, thanks!